src/DynamoCore/Extensions/PackageInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Extensions
+{
+    /// <summary>
+    /// Class containing info about a Dynamo package. 
+    /// Used for serialization.
+    /// </summary>
+    public class PackageInfo
```

Comment:
seems like an odd namespace for this class?

Reply:
Agreed. I moved it to `Dynamo.Graph.Workspaces`. Also thinking of renaming it `PackageDependency`, which might make more sense in the context of workspaces.